core/src/it/scala/org/ensime/fixture/ProjectFixture.scala

```diff
@@ -58,12 +58,19 @@ object ProjectFixture extends Matchers {
         Broadcaster.Persist(AnalyzerReadyEvent),
         Broadcaster.Persist(IndexerReadyEvent)
       )
-    else
-      probe.receiveN(3, 2.minutes.dilated) should contain only (
-        Broadcaster.Persist(AnalyzerReadyEvent),
-        Broadcaster.Persist(FullTypeCheckCompleteEvent),
-        Broadcaster.Persist(IndexerReadyEvent)
-      )
+    else {
+      type Fish = PartialFunction[Any, Boolean]
+      val fishFor3Messages: Fish = {
+        case Broadcaster.Persist(AnalyzerReadyEvent) => true
+        case Broadcaster.Persist(FullTypeCheckCompleteEvent) => true
+        case Broadcaster.Persist(IndexerReadyEvent) => true
+        case _ => false
+      }
+
+      probe.fishForMessage(2.minutes)(fishFor3Messages)
```

Comment:
hmm, not too keen on this change. We need to assert that it is those exact three messages that we get. Maybe there is another way to do it (and don't forget to dilate).